posnums.c

```diff
@@ -0,0 +1,89 @@
+/* This program allows a user to enter two numbers, which will be identifiedas either prime or composite individually and relatively prime as a pair. Also, the averages of these two numbers will be printed and whether it is prime or composite. - Amanda Cuevas */
```

Comment:
Perfect job! You're actually one of the only people who got this completely correct (passed all the edge cases). Superb job! Loved the touch of telling us whether the average was prime or not. 💯